File: archunit/solution/ArchUnit-kata/Layered/Services/SuperHeroService.cs
using ArchUnit.Kata.Layered.Repositories;

namespace ArchUnit.Kata.Layered.Services
{
    public class SuperHeroService
    {
        private readonly ISuperHeroRepository superHeroRepository;

        public SuperHeroService(ISuperHeroRepository superHeroRepository)
        {
            this.superHeroRepository = superHeroRepository;
        }
    }
}

File: archunit/solution-java/src/main/java/archunit/layered/service/SuperHeroService.java
package archunit.layered.service;

import archunit.layered.domain.SuperHeroRepository;

public class SuperHeroService {
    private final SuperHeroRepository superHeroRepository;

    public SuperHeroService(SuperHeroRepository superHeroRepository) {
        this.superHeroRepository = superHeroRepository;
    }

    public SuperHeroRepository getSuperHeroRepository() {
        return superHeroRepository;
    }
}
